drivers/hmis/app/models/hmis/user.rb

```diff
@@ -244,6 +244,13 @@ def pick_list_group_label
     end
   end
 
+  def to_pick_list_hash
```

Comment:
We lave lots of existing methods like this called `def to_pick_list_option`, prefer updating this to consistent name